src/main/resources/messages.properties

```diff
@@ -19,7 +19,7 @@ footerlinks = Links
 country = Country
 dod = Day of departure
 nop = Number of people
-nod = Number of days:
+ad = Arrival day
```

Comment:
ad? what is it???

Reply:
AD - arrival day. Main page got used to changes. Switched input field for number of days on date picker.